ccf_specs.md

```diff
@@ -562,12 +588,19 @@ ccf-composite-type-message =
             / contract-type
             / event-type
             / enum-type
+            / struct-interface-type
+            / resource-interface-type
+            / contract-interface-type
         )
     ])
 
-composite-type = [
+composite-type-id = (
     id: bstr,
     location-identifier: tstr,
```

Comment:
What are `id` and `location-identifier`? Is `id` the internal ID used for de-duplicating type information in this encoding, or Cadence "type ID" (location + qualified identifier, e.g. `A.0000000000000001.SomeContract.SomeResource`)?

Reply:
That makes sense! Thank you for the clarification 👍